docs/tasks/uninstall/kubernetes.rst

```diff
@@ -0,0 +1,85 @@
+==========================
+Uninstalling on Kubernetes
+==========================
+
+Below is the processes for uninstalling cert-manager on Kubernetes. There are
+two processes to chose depending on which method you used to install
+cert-manager - static manifests or ``helm``.
+
+.. warning::
+
+   To uninstall cert-manger you should always use the same process for installing
+   but in reverse. Deviating from the following process whether cert-manager has
+   been installed from static manifests or helm can cause issues and
+   potentially broken states. Please ensure you follow the below steps when
+   uninstalling to prevent this happening.
+
+Before continuing, ensure that all cert-manager resources that have been created
+by users have been deleted. You can check for any existing resources with the
+following command:
+
+.. code-block:: shell
+
+   kubectl get Issuers,ClusterIssuers,Certificates,CertificateRequests --all-namespaces
+
+Once all these resources have been deleted you are ready to uninstall
+cert-manager using the procedure determined by how you installed.
+
+Uninstalling with regular manifests
+===================================
+
+Uninstalling from an installation with regular manifests is a case of running
+the installation process, *in reverse*, using the delete command of ``kubectl``.
+
+First delete the installation manifests using a link to your currently running
+version vX.Y.Z like so:
+
+.. code-block:: shell
+
+   kubectl delete -f https://github.com/jetstack/cert-manager/releases/download/vX.Y.Z/cert-manager.yaml
+
+Finally, delete the cert-manager namespace:
+
+.. code-block:: shell
+
+   kubectl delete namespace cert-manager
```

Comment:
This should be done automatically if the user used the above link to *install* I think 😄